home/serializers.py

```diff
@@ -665,3 +845,86 @@ def to_representation(self, page):
 class AssessmentSerializer(BaseSerializer):
     locale = PageLocaleField(read_only=True)
     questions = QuestionField(read_only=True)
+
+
+class TemplateLocaleField(serializers.Field):
+    """
+    Serializes the "locale" field.
+    """
+
+    def get_attribute(self, instance):
+        return instance
+
+    def to_representation(self, instance):
+        return instance.locale.language_code
+
+
+class MessageField(serializers.Field):
+    """
+    Serializes the "message" field.
+    """
+
+    def get_attribute(self, instance):
+        return instance
+
+    def to_representation(self, instance):
+        return instance.message
```

Comment:
These all seem like quite simple serializations. Could we not just use something like a `CharField`, or does the base ModelSerializer not just do this correctly for us, and we can just specify them in the `fields` list?